code/wdb/datastripe.q

```diff
@@ -36,6 +36,9 @@ modaccess:{[accesstab]};
 
     };
 
+//Function to check if a segid is defined if datastriping is on
+checksegid:{if[.ds.datastripe;if[not (`segid in key .proc.params);.lg.e[`init;"Datastriping is turned on however no segment id has been defined for this process"]]]}
```

Comment:
this appears to be the same function defined in two places - can we not put it in some common code location rather than having two definitions?